rune/dist/centos/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: rpm release-tarball
+
+RPMBUILD_DIR := $(shell mktemp -u /tmp/rpmbuild.XXXX)
+INCLAVARE_CONTAINERS_VERSION := $(shell cat ../../../VERSION)
```

Comment:
This variable should be defined again and again. Re-defining it in rune is useful because rune will be available on another repo, but rune/dist/centos/ is never. So INCLAVARE_CONTAINERS_VERSION should be exported from the top of rune directory and don't define it again.